### PR TITLE
refactor(checkout): PI-2349 moved Worldpay Access payment strategy to…

### PR DESCRIPTION
**Pull request by vitalii-koshovyi**
… separate package

## What?
Moved Worldpay Access payment strategy

## Why?
Due to the monorepo refactoring

## Testing / Proof
Units/manually

3ds challange

https://github.com/bigcommerce/checkout-sdk-js/assets/79574476/da041d20-f0ac-4b65-a241-e3823fcda390

Successful Authentication (Frictionless)

https://github.com/bigcommerce/checkout-sdk-js/assets/79574476/22866ebb-e606-4150-866d-9a2b4ab91cb3

@bigcommerce/team-checkout @bigcommerce/team-payments

### CODE REVIEW

File: packages/worldpayaccess-integration/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# worldpayaccess-integration
+
+This library was generated with [Nx](https://nx.dev).
```
Comment (animesh1987) [inline packages/worldpayaccess-integration/README.md:3]: Can we have a more informative readme please.

File: packages/core/src/payment/strategies/worldpayaccess/index.ts [hunk 2]
```diff
@@ -1,2 +0,0 @@
-export { WorldpayAccessPaymentInitializeOptions } from './worldpayaccess-payment-options';
-export { default as WorldpayaccessPaymetStrategy } from './worldpayaccess-payment-strategy';
```
Comment (bc-peng) [inline packages/core/src/payment/strategies/worldpayaccess/index.ts:2]: Do we also need to remove those files?

**Review by serhii-tkachenko — APPROVED** [review]
Looks good to me 👍

File: packages/core/src/payment/strategies/worldpayaccess/index.ts (hunk 2 above)
Comment (vitalii-koshovyi) [reply packages/core/src/payment/strategies/worldpayaccess/index.ts:2]: yes, sure

Comment (vitalii-koshovyi) [reply packages/core/src/payment/strategies/worldpayaccess/index.ts:2]: I've removed them already

File: packages/worldpayaccess-integration/project.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            "options": {
+                "lintFilePatterns": ["packages/worldpayaccess-integration/**/*.ts"]
+            }
+        },
+        "test": {
+            "executor": "@nrwl/jest:jest",
+            "outputs": ["coverage/packages/worldpayaccess-integration"],
+            "options": {
+                "jestConfig": "packages/worldpayaccess-integration/jest.config.js",
+                "passWithNoTests": true
```
Comment (animesh1987) [inline packages/worldpayaccess-integration/project.json:18]: Just saw this, do we need this?

Comment (vitalii-koshovyi) [reply packages/worldpayaccess-integration/project.json:18]: I'll remove it, it was generated

Comment (vitalii-koshovyi) [reply packages/worldpayaccess-integration/project.json]: but all packages have this

Comment (animesh1987) [reply packages/worldpayaccess-integration/project.json]: 🤦 maybe need to go and update for all packages.

**Review by animesh1987 — APPROVED** [review]

**Pull request merged**